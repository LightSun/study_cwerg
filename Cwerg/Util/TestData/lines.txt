" ä ß "
"\"\""
"\n\n\"\"\\"
"\0\00\122\3"
"\x00\x01\x08\x10\x09\x02\x03\n\x11\x18 \x19\x12\x0b\x04\x05\x0c\x13\x1a!(0)\"\x1b\x14\x0d\x06\x07\x0e\x15\x1c#*1892+$\x1d\x16\x0f\x17\x1e%,3:;4-&\x1f'.5<=6/7>?"
"\x00\x00\x80?"
a b c d 11
[]
"a b" c "d 11"
a b [c d] 11
"a b" [c "d 11"]
"a b" [] c "d 11"
"a b" [a [b]]
"  # "
" # " # "  # "
# "aaa"
# "a
#,;;
"aaaaa" # sss
a , b
"jjjj
[aa 
][
int64 -1000
int64 -9223372036854775808
int64 0x8000000000000000
int64 0xffffffffffffffff
int64 4.5
int64 +nan
int64 0.0
int64 -0.0
uint64 1000
uint64 9223372036854775808
uint64 0x8000000000000000
uint64 0xffffffffffffffff
uint64 4.5
uint64 +nan
uint64 0.0
uint64 -0.0
flt64 4.5
flt64 0.25
flt64 nan
flt64 +nan
flt64 -inf
flt64 -INF
flt64 .0
flt64 1e4
flt64 +inf
flt64 0.0
flt64 -0.0
flt64 0x7ff8000000000000
flt64 0x0
flt64 0x3fd0000000000000
flt64 0.25
flt64 .0009765625

