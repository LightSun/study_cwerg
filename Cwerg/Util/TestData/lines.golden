
" ä ß "
" \xc3\xa4 \xc3\x9f "

"\"\""
"\"\""

"\n\n\"\"\\"
"\n\n\"\"\\"

"\0\00\122\3"
"\x00\x00R\x03"

"\x00\x01\x08\x10\x09\x02\x03\n\x11\x18 \x19\x12\x0b\x04\x05\x0c\x13\x1a!(0)\"\x1b\x14\x0d\x06\x07\x0e\x15\x1c#*1892+$\x1d\x16\x0f\x17\x1e%,3:;4-&\x1f'.5<=6/7>?"
"\x00\x01\x08\x10\x09\x02\x03\n\x11\x18 \x19\x12\x0b\x04\x05\x0c\x13\x1a!(0)\"\x1b\x14\x0d\x06\x07\x0e\x15\x1c#*1892+$\x1d\x16\x0f\x17\x1e%,3:;4-&\x1f'.5<=6/7>?"

"\x00\x00\x80?"
"\x00\x00\x80?"

a b c d 11
{a} {b} {c} {d} {11}

[]
{[} {]}

"a b" c "d 11"
"a b" {c} "d 11"

a b [c d] 11
{a} {b} {[} {c} {d} {]} {11}

"a b" [c "d 11"]
"a b" {[} {c} "d 11" {]}

"a b" [] c "d 11"
"a b" {[} {]} {c} "d 11"

"a b" [a [b]]
@FAILED@

"  # "
"  # "

" # " # "  # "
" # " {# "  # "}

# "aaa"
{# "aaa"}

# "a
{# "a}

#,;;
{#,;;}

"aaaaa" # sss
"aaaaa" {# sss}

a , b
@FAILED@

"jjjj
@FAILED@

[aa 
@FAILED@

][
@FAILED@

int64 -1000
{int64} {-1000}
[INT64] -1000 fffffffffffffc18

int64 -9223372036854775808
{int64} {-9223372036854775808}
[INT64] -9223372036854775808 8000000000000000

int64 0x8000000000000000
{int64} {0x8000000000000000}
[INT64] -9223372036854775808 8000000000000000

int64 0xffffffffffffffff
{int64} {0xffffffffffffffff}
[INT64] -1 ffffffffffffffff

int64 4.5
{int64} {4.5}
[INT64] @BAD VALUE@

int64 +nan
{int64} {+nan}
[INT64] @BAD VALUE@

int64 0.0
{int64} {0.0}
[INT64] @BAD VALUE@

int64 -0.0
{int64} {-0.0}
[INT64] @BAD VALUE@

uint64 1000
{uint64} {1000}
[UINT64] 1000 3e8

uint64 9223372036854775808
{uint64} {9223372036854775808}
[UINT64] 9223372036854775808 8000000000000000

uint64 0x8000000000000000
{uint64} {0x8000000000000000}
[UINT64] 9223372036854775808 8000000000000000

uint64 0xffffffffffffffff
{uint64} {0xffffffffffffffff}
[UINT64] 18446744073709551615 ffffffffffffffff

uint64 4.5
{uint64} {4.5}
[UINT64] @BAD VALUE@

uint64 +nan
{uint64} {+nan}
[UINT64] @BAD VALUE@

uint64 0.0
{uint64} {0.0}
[UINT64] @BAD VALUE@

uint64 -0.0
{uint64} {-0.0}
[UINT64] @BAD VALUE@

flt64 4.5
{flt64} {4.5}
[FLT64] 4.5 4012000000000000

flt64 0.25
{flt64} {0.25}
[FLT64] 0.25 3fd0000000000000

flt64 nan
{flt64} {nan}
[FLT64] nan 7ff8000000000000

flt64 +nan
{flt64} {+nan}
[FLT64] nan 7ff8000000000000

flt64 -inf
{flt64} {-inf}
[FLT64] -inf fff0000000000000

flt64 -INF
{flt64} {-INF}
[FLT64] -inf fff0000000000000

flt64 .0
{flt64} {.0}
[FLT64] 0 0

flt64 1e4
{flt64} {1e4}
[FLT64] 10000 40c3880000000000

flt64 +inf
{flt64} {+inf}
[FLT64] inf 7ff0000000000000

flt64 0.0
{flt64} {0.0}
[FLT64] 0 0

flt64 -0.0
{flt64} {-0.0}
[FLT64] -0 8000000000000000

flt64 0x7ff8000000000000
{flt64} {0x7ff8000000000000}
[FLT64] nan 7ff8000000000000

flt64 0x0
{flt64} {0x0}
[FLT64] 0 0

flt64 0x3fd0000000000000
{flt64} {0x3fd0000000000000}
[FLT64] 0.25 3fd0000000000000

flt64 0.25
{flt64} {0.25}
[FLT64] 0.25 3fd0000000000000

flt64 .0009765625
{flt64} {.0009765625}
[FLT64] 0.000976562 3f50000000000000



